test/easyconfigs/easyconfigs.py

```diff
@@ -297,6 +296,14 @@ def template_easyconfig_test(self, spec):
     app.close_log()
     os.remove(app.logfile)
 
+    # dump the easyconfig file
+    ec.dump('test.eb')
+    dumped_ec = EasyConfig('test.eb')
```

Comment:
don't create files in the current directory, use a temporary path instead

and clean it up once you're done with it:

```python
test_ecfile, handle = tempfile.mkstemp()
os.close(handle)

ec.dump(test_ecfile)
dumped_ec = EasyConfig(test_ecfile)
os.remove(test_ecfile)
```